taxi_service/urls.py

```diff
@@ -14,8 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path('taxi/', include('taxi.urls', namespace='taxi'))
 ]
```

Comment:
Add path to serve static files, use function [static()](https://docs.djangoproject.com/en/4.0/ref/urls/#django.conf.urls.static.static)